package.json

```diff
@@ -47,10 +47,5 @@
     ],
     "*.{jsx,js}": "eslint --cache --fix",
     "*.{js,css,md}": "prettier --write"
-  },
-    "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
```

Comment:
Please undo this deletion

Reply:
@seifeldeen3388 
it doesn't enable me to push my work[](url)`
![img2](https://user-images.githubusercontent.com/14257006/104105139-e8e75e00-52b4-11eb-998d-792587c70df0.PNG)
`

Reply:
![img2](https://user-images.githubusercontent.com/14257006/104105154-ff8db500-52b4-11eb-8ebc-42f3cebe9275.PNG)

Reply:
Yes, delete it without committing.
Just temporarily until the issue is fixed.